### PR TITLE
SSO For Non-Browser Applications

### PR DESCRIPTION
**Pull request by narthollis**
Detail common methods for performing OAuth2 workflows in a non-browser environment.

### CODE REVIEW

**Comment by JimiC** [comment]
Wow. Impressive.
You may wonna do another commit though and add your name in the CONTRIBUTORS file.

File: docs/sso/nonbrowserapps.md [hunk 1]
```diff
@@ -1,4 +1,93 @@
...
+
+## Embedded Browser
+**This method is not recommended as it breaks the trust separation between application and authentication provider.
+It is detailed here for completeness, and should not be used unless the above options are not feasible.**
+
+### Pros
+* Works everywhere all the time
+
+### Cons
+* THe user will be entering their EVE credentials inside your application, breaking the trust separation.
```
Comment (JimiC) [inline docs/sso/nonbrowserapps.md]: Fix the capitalization of the word 'THe'.

**Comment by narthollis** [comment]
As it turns out I forgot a method (manual copy/paste) that I want to add...

**Comment by Regner** [comment]
Thank you very much. :)

**Pull request merged**